tests/unit/btree_stress_test.c

```diff
@@ -29,13 +29,12 @@
 typedef struct insert_thread_params {
    cache           *cc;
    btree_config    *cfg;
-   platform_heap_id heap_id;
+   platform_heap_id hid;
    btree_scratch   *scratch;
    mini_allocator  *mini;
    uint64           root_addr;
    int              start;
    int              end;
-   platform_heap_id hid;
```

Comment:
There are two instances of this variable, causing confusion below -- we ended up using one for allocation and one for free.

(This is probably a bug that sneaked-in when I initially ported these tests to C-Unit framework.)